vision/object_detection_segmentation/yolov2-coco/yolov2_coco_md5.txt

```diff
@@ -0,0 +1 @@
+MD5 (yolov2_coco.onnx) = dd60e6fb27a5956b7705c985e1f62737
```

Comment:
Can you format this as:
dd60e6fb27a5956b7705c985e1f62737 yolov2_coco.onnx

Reply:
Thank you for your comment. I changed it